include/webui.h

```diff
@@ -50,6 +50,11 @@
 
 // -- Windows -------------------------
 #ifdef _WIN32
+    #ifndef WINVER
+        // set minimum windows version requirement (windows 10, version 1607)
+        // (To use DPI awareness functions, 'SetProcessDpiAwarenessContext'...)
+        #define WINVER 0x0605
```

Comment:
Where does this Windows version limitation come from? If it's just for the DPI awareness, it seems unreasonable to limit Windows version for the entire library.

Reply:
No limit on type of windows, 7,8,10,11 what ever.
Actually it just related on build version 1703, old windows will need to be updated.

Refer this(win7 need to update win7.sp1.) : https://github.com/AvaloniaUI/Avalonia/issues/2095

And current CI's zig build results of all possible windows type are passed, right?

Reply:
The CI only contains one version of Windows. There is indeed a limit on the version of Windows, as the `SetProcessDpiAwarenessContext` function is only available on `Windows 10, version 1703` and higher. Wouldn't it be better to use the older `SetProcessDpiAwareness` to support up to `Windows 8.1`?

Reply:
You're right, I misunderstood.
I fixed it up like this.
```cpp
// Windows DPI awareness support
#ifdef _WIN32
#include <shellscalingapi.h>	// for 'SetProcessDpiAwareness' function use
static void __InitDpiAwareness(void) {
	{	// minimum platform : Windows 10, version 1703 / Windows Server 2016
		BOOL (*pSetProcessDpiAwarenessContext)(DPI_AWARENESS_CONTEXT value) = (BOOL (*)(DPI_AWARENESS_CONTEXT))GetProcAddress(GetModuleHandle("user32.dll"), "SetProcessDpiAwarenessContext");

		if (pSetProcessDpiAwarenessContext) {
			pSetProcessDpiAwarenessContext(DPI_AWARENESS_CONTEXT_PER_MONITOR_AWARE_V2);
			return;
		}
	}
	{	// minimum platform : Windows 8.1 / Windows Server 2012 R2
		HMODULE hLib = LoadLibrary("Shcore.dll");	// dynamic load lib is required.
		if (hLib) {
			HRESULT (*pSetProcessDpiAwareness)(PROCESS_DPI_AWARENESS value) = (HRESULT (*)(PROCESS_DPI_AWARENESS))GetProcAddress(hLib, "SetProcessDpiAwareness");
			if (pSetProcessDpiAwareness) {
				if (pSetProcessDpiAwareness(PROCESS_PER_MONITOR_DPI_AWARE) != S_OK) {
					pSetProcessDpiAwareness(PROCESS_SYSTEM_DPI_AWARE);
				}
				return;
			}
		}
	}
	{	// minimum platform : Windows Vista / Windows Server 2008
		BOOL (*pSetProcessDPIAware)() = (BOOL (*)())GetProcAddress(GetModuleHandle("user32.dll"), "SetProcessDPIAware");
		if (pSetProcessDPIAware) {
			pSetProcessDPIAware();
		}
	}
	
	// If it reached to here, what ancient Windows is used?
}

static UINT __GetDpiForWindow(HWND hwnd) {
	static bool bInitialized	= false;
	static UINT (*pGetDpiForWindow)(HWND hwnd) = NULL;
	
	if (!bInitialized) {
		pGetDpiForWindow = (UINT (*)(HWND))GetProcAddress(GetModuleHandle("user32.dll"), "GetDpiForWindow");
		bInitialized = true;
	}
	// if it's supported
	if(pGetDpiForWindow) return pGetDpiForWindow(hwnd);
	// or not, return default DPI value. (under a Win 10.)
	return USER_DEFAULT_SCREEN_DPI;
}
#endif
```
Same problem with `GetDpiForWindows()` is fixed.

Reply:
The change looks great! Thank you for your work.